tests/e2e/fastfailover_test.go

```diff
@@ -87,6 +91,10 @@ var _ = Describe("Fast failover", Serial, Label(tests.LabelPerformance), func()
 	})
 
 	Context("with async replicas cluster", func() {
+		if strings.Contains(os.Getenv("POSTGRES_IMG"), ":10") {
+			// Cluster file without replication slot since it requires PostgreSQL 11 or above
+			sampleFile = sampleFileWithoutReplSlots
+		}
```

Comment:
I would prefer to have the PostgreSQL version in the env structure, so we can detect it in a centralized and controllable way.

Reply:
Resolved.